base_user_role/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "User roles",
-    "version": "15.0.0.3.4",
+    "version": "15.0.0.3.5",
```

Comment:
no need to bump the version, it will be done automatically when merging. 
(and it can avoid conflict).

Reply:
That's a good trick. I'll have to see about adding it to the pre-commit of my own modules.